imports/ui/components/register/client/register.tests.js

```diff
@@ -0,0 +1,100 @@
+import {
+    name as Register
+} from '../register';
+
+import 'angular-mocks';
+
+import {
+    Meteor
+} from 'meteor/meteor';
+import {
+    Accounts
+} from 'meteor/accounts-base';
+
+import {
+    sinon
+} from 'meteor/practicalmeteor:sinon';
+
+describe('Register', function() {
+    spies.restoreAll();
+
+    if (!process.env.TESTING)
+        process.env.TESTING = 1;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '{' and instead saw 'process'.](https://www.codacy.com/app/ddspog/MAM-Seed/file/2729684824/issues/source?bid=3340041&fileBranchId=3346427#l22)